### PR TITLE
Update `CellExplorerSortingInterface` to new format

### PR DESCRIPTION
**Pull request by h-mayorquin**
As we have working on https://github.com/catalystneuro/buzsaki-lab-to-nwb/issues/55 we have discussed with one of the members of the Buzsaki lab which shared the new specification with us. In the new specification the samping_rate is no longer present in a file `sessionInfo` as it used to be. I am updating both this interface and the sorting extractor in spikeinterface to read the sampling frequency when is available in the same file as the new format stipulates

https://github.com/SpikeInterface/spikeinterface/pull/1628

Note quite suere if I am setting the requirements right to test.

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
Biggest comment will be similar to the extractor - I would try to maintain it as a file path style unless there is a significant reason to swap to folder; the only difference I'm aware of ATM is the different type of header file from previous versions

**Comment by h-mayorquin** [comment]
Yeah, mabye you are right, @CodyCBakerPhD .

I am also thinking on loading the properties in spikeinterface instead of doing that here.

**Comment by CodyCBakerPhD** [comment]
> I am also thinking on loading the properties in spikeinterface instead of doing that here.

I would be OK with that too, good idea. Kind of similar to what I did for the `StreaingIblSortingExtractor` (or w/e it's called)

**Comment by h-mayorquin** [comment]
> > I am also thinking on loading the properties in spikeinterface instead of doing that here.
> 
> I would be OK with that too, good idea. Kind of similar to what I did for the `StreaingIblSortingExtractor` (or w/e it's called)

Going to do that later though. Once we are checking the data of Huszar conversion in detail. I am hoping I will be more familiar with the data and I will be able to make a better design decision at that point.

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin Could you send me an example file for this? Just for a sanity check

File: src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def __init__(self, file_path: FilePathType, verbose: bool = True):
 
         hdf5storage = get_package(package_name="hdf5storage")
 
-        super().__init__(spikes_matfile_path=file_path, verbose=verbose)
+        super().__init__(file_path=file_path, verbose=verbose)
```
Comment (CodyCBakerPhD) [inline src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py:37]: Oh, also, issue a deprecation warning on NeuroConv side

Comment (h-mayorquin) [reply src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py:37]: Good idea. 

Here is the file:
https://www.dropbox.com/s/q4i4dsnk7rqh09h/Peter_MS22_180629_110319_concat.spikes.cellinfo.mat?dl=0

Maybe you can drop the other columns to make it smaller so we can add it to the gin tests?

Comment (h-mayorquin) [reply src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py:53]: Wait, I did something possibly dumb here. Can you check out the new code and see if this is what you intended? You already had the right input `file_path` and that is only handled internally.

Comment (CodyCBakerPhD) [reply src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py:53]: Sorry, I mean the converted NWB file from using this interface (possibly on that file, but also maybe other GIN files too)

Comment (h-mayorquin) [reply src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py:53]: What did you mean about the deprecation though? What should be deprecated?

**Comment by h-mayorquin** [comment]
Not necessary any more after this is done inside spikeinterface now.

**Pull request closed without merging**